### ISSUE TITLE
Webpack 5 release announcement

### ISSUE THREAD

**Issue by srittau** [webpack/webpack.js.org#7262]
When I go to webpack.js.org, I get a popup in the lower left corner saying:

> Webpack 5 has been officially released. Read our [announcement](https://webpack.js.org/blog/2020-10-10-webpack-5-release/). Not ready yet? Read [webpack 4 documentation here](https://v4.webpack.js.org/).

Considering that Webpack 5 has been released 3.5 years ago, maybe it's time to retire that notice?

**Comment by alexander-akait** [comment]
Some people still use webpack 4 :disappointed:

**Comment by srittau** [comment]
In that case, maybe reword it a bit? Mentioning the release announcement makes it sound as if this is a fairly recent development. Maybe something along the lines of:

> You are reading the documentation for webpack 5, the latest released version. Read the [webpack 4 documentation here](https://v4.webpack.js.org/). See the [migration guide](https://webpack.js.org/migrate/5) for upgrading to webpack 5.

**Comment by alexander-akait** [comment]
@srittau Do you want to send a PR, I am fine with it, thank you
